wp-content/themes/bootstrap/functions.php

```diff
@@ -45,7 +46,38 @@ function register_widgets(){
                      ]);
 }
 
+function register_events_post_type():void
+{
+    register_post_type('event', [
+        'label' => 'Events',
+        'labels' => [
+            'name' => 'Events',
+            'singular_name' => 'event',
+            'add_new' => 'add new event',
 
+        ],
+        'public' => true,
+        'rewrite' => ['slug' => 'events'],
+        'has_archive' => true,
+        'show_in_rest' => true,
+        'supports' => ['title', 'editor', 'thumbnail', 'comments'],
+        'menu_icon' => 'dashicons-smiley',
+    ]);
+
+    register_taxonomy('event_category', ['event'], [
+        'labels' => [
+            'name' => 'Event Category',
+            'singular_name' => 'event category',
+            'add_new' => 'add new event category',
+        ],
+        'hierarchical' => false,
+        'show_ui' => true,
+        'show_in_rest' => true,
+        'show_admin_column' => true,
+        'query_var' => true,
+        'rewrite' => ['slug' => 'event category'],
```

Comment:
slug обычно не содержит пробелов
```suggestion
        'rewrite' => ['slug' => 'event-category'],
```